client/asset/btc/btc.go

```diff
@@ -945,21 +960,29 @@ func (a amount) String() string {
 	return strconv.FormatFloat(btcutil.Amount(a).ToBTC(), 'f', -1, 64) // dec, but no trailing zeros
 }
 
-// feeRateWithFallback attempts to get the optimal fee rate in sat / byte via
-// FeeRate. If that fails, it will return the configured fallback fee rate.
-func (btc *ExchangeWallet) feeRateWithFallback(confTarget, feeSuggestion uint64) uint64 {
+// targetFeeRateWithFallback attempts to get a fresh fee rate for the target
+// number of confirmations, but falls back to the suggestion or fallbackFeeRate
+// via feeRateWithFallback.
+func (btc *ExchangeWallet) targetFeeRateWithFallback(confTarget, feeSuggestion uint64) uint64 {
 	feeRate, err := btc.estimateFee(btc.node, confTarget)
 	if err == nil {
 		btc.log.Tracef("Obtained local estimate for %d-conf fee rate, %d", confTarget, feeRate)
 		return feeRate
 	}
-	if feeSuggestion > 0 && feeSuggestion < btc.fallbackFeeRate && feeSuggestion < btc.feeRateLimit {
```

Comment:
Noting that I've removed the `feeSuggestion < btc.fallbackFeeRate` filter. I'm not sure what the reasoning was there, but I can't see why our "fallback" rate would also be used an an upper limit. We already have a `feeRateLimit`.